deployment/RS03ASHS_Deploy.csv

```diff
@@ -9,5 +9,8 @@ TN-326,,,,RS03ASHS-PN03B-06-CAMHDA301,2,1,2015-07-09T13:28:24,2016-07-26T21:18:0
 SKQ201610S,,,,RS03ASHS-PN03B-06-CAMHDA301,3,1,2016-07-27T05:49:19,2017-08-13T23:30:00,ATSOO-70232-01H3B,,ATAPL-70168-00001,45.93364,-130.01364,,1542,1542,
 RR1714,,,,RS03ASHS-MJ03B-07-TMPSFA301,3,1,2017-08-15T00:00:00,,ATAPL-65244-050-0027,,ATAPL-69971-00002,45.93362,-130.01368,,1540,1540,
 RR1714,,,,RS03ASHS-MJ03B-09-BOTPTA304,3,1,2017-08-15T00:00:00,,ATAPL-65244-050-0027,,ATAPL-58316-00005,45.93363,-130.01368,,1542,1542,
-RR1714,,,,RS03ASHS-MJ03B-10-CTDPFB304,3,1,2017-08-15T00:00:00,,ATAPL-65244-050-0027,,ATCWK-67627-00006,45.93363,-130.01368,,1542,1542,
-RR1714,,,,RS03ASHS-PN03B-06-CAMHDA301,4,1,2017-08-14T07:00:00,,ATSOO-70232-01H3B,,ATAPL-70168-00002,45.93362,-130.01364,,1542,1542,
+RR1714,,,,RS03ASHS-MJ03B-10-CTDPFB304,3,1,2017-08-15T00:00:00,2018-07-04T00:00:00,ATAPL-65244-050-0027,,ATCWK-67627-00006,45.93363,-130.01368,,1542,1542,
```

Comment:
`ERROR:     Type mismatch for sensor.uid - expected "Sensor", found "Platform" - row 12`

Probably missing a second comma after the second date.

Reply:
It shows up in a table aligned with the other columns.